File: html_code/js/common/code.js
const Code = {
	/** 删除标志-禁用 */
	"DEL_FLG_0": {"group":"del_flg","name":"禁用","value":"0"},
	/** 删除标志-启用 */
	"DEL_FLG_1": {"group":"del_flg","name":"启用","value":"1"},
	
	/** 平台- 全部*/
	"PLATFORM_WEB_ALL": {"group":"platform","name":"All","value":"0"},
	/** 平台-Window */
	"PLATFORM_WEB_WINDOW": {"group":"platform","name":"Window","value":"1"},
	/** 平台-Mac */
	"PLATFORM_APP_MAC": {"group":"platform","name":"Mac","value":"2"},
	/** 平台-Android */
	"PLATFORM_APP_ANDROID": {"group":"platform","name":"Android","value":"3"},
	/** 平台-IOS */
	"PLATFORM_APP_IOS": {"group":"platform","name":"IOS","value":"4"},
	
	/** 请求方式-全部 */
	"REQUEST_METHOD_ALL": {"group":"request_method","name":"all method","value":"0"},
	/** 请求方式-get */
	"REQUEST_METHOD_GET": {"group":"request_method","name":"get","value":"1"},
	/** 请求方式-post */
	"REQUEST_METHOD_POST": {"group":"request_method","name":"post","value":"2"},
	/** 请求方式-delete */
	"REQUEST_METHOD_DELETE": {"group":"request_method","name":"delete","value":"3"},
	/** 请求方式-put */
	"REQUEST_METHOD_PUT": {"group":"request_method","name":"put","value":"4"},
	
	/** 否-false */
	"NO": {"group":"yes_no","name":"否","value":"0"},
	/** 是-true */
	"YES":{"group":"yes_no","name":"是","value":"1"},
	
	/** 禁用 */
	"disable": {"group":"use_type","name":"禁用","value":"0"},
	/** 启用 */
	"enable":{"group":"use_type","name":"启用","value":"1"}
}
$.deepFreeze(Code);

File: web_code/Taurus/src/main/java/org/taurus/common/code/Code.java
package org.taurus.common.code;

public enum Code {

	/** 删除标志-禁用 */
	DEL_FLG_0("del_flg", "禁用", "0"),
	/** 删除标志-启用 */
	DEL_FLG_1("del_flg", "启用", "1"),

	/** 平台- 全部 */
	PLATFORM_WEB_ALL("platform", "All", "0"),
	/** 平台-Window */
	PLATFORM_WEB_WINDOW("platform", "Window", "1"),
	/** 平台-Mac */
	PLATFORM_APP_MAC("platform", "Mac", "2"),
	/** 平台-Android */
	PLATFORM_APP_ANDROID("platform", "Android", "3"),
	/** 平台-IOS */
	PLATFORM_APP_IOS("platform", "IOS", "4"),

	/** 请求方式-全部 */
	REQUEST_METHOD_ALL("request_method", "all method", "0"),
	/** 请求方式-GET */
	REQUEST_METHOD_GET("request_method", "get", "1"),
	/** 请求方式-POST */
	REQUEST_METHOD_POST("request_method", "post", "2"),
	/** 请求方式-DELETE */
	REQUEST_METHOD_DELETE("request_method", "delete", "3"),
	/** 请求方式-PUT */
	REQUEST_METHOD_PUT("request_method", "put", "4"),

	/** 否-false */
	NO("yes_no", "否", "0"),
	/** 是-true */
	YES("yes_no", "是", "1"),

	/** 文件类型-其它*/
	FILE_TYPE_OTHER("file_type", "other", "0"),
	/** 文件类型-音频*/
	FILE_TYPE_AUDIO("file_type", "audio", "1"),
	/** 文件类型-视频*/
	FILE_TYPE_VIDEO("file_type", "video", "2"),
	/** 文件类型-文本*/
	FILE_TYPE_TXT("file_type", "txt", "3"),
	/** 文件类型-日志*/
	FILE_TYPE_LOG("file_type", "log", "4"),
	/** 文件类型-图片*/
	FILE_TYPE_PICTURE("file_type", "picture", "5"),

	/** 日志处理状态-待处理*/
	LOG_STATUS_0("log_status","待处理","0"),
	/** 日志处理状态-已处理*/
	LOG_STATUS_1("log_status","已处理","1"),
	/** 日志处理状态-无需处理*/
	LOG_STATUS_2("log_status","无需处理","2"),

	/** 资源类别-系统资源 */
	RESOURCE_TYPE_SYSTEM("resource_type","系统资源","-1"),
	/** 资源类别-用户根目录 */
	RESOURCE_TYPE_ROOT("resource_type","用户根目录","0"),
	/** 资源类别-自定义资源 */
	RESOURCE_TYPE_CUSTOM("resource_type","自定义资源","1");

	private String group;

	private String name;

	private String value;

	private Code(String group, String name, String value) {
		this.group = group;
		this.name = name;
		this.value = value;
	}

	/**
	 * 获取code分组
	 * 
	 * @return
	 */
	public String getGroup() {
		return group;
	}

	/**
	 * 获取code名称
	 * 
	 * @return
	 */
	public String getName() {
		return name;
	}

	/**
	 * 获取code值
	 * 
	 * @return
	 */
	public String getValue() {
		return value;
	}

}
